test/functorch/test_control_flow.py

```diff
@@ -1174,6 +1203,225 @@ def fwbw(map_op, f, x, y):
         fake_outs = fwbw(_fake_map, f, x, y)
         self.assertEqual(true_outs, fake_outs)
 
+    @unittest.skipIf(not torch.cuda.is_available(), "Test requires CUDA.")
```

Comment:
We could add use the `parametrize` decorator for reverse=True/False when applicable.

Reply:
Done